makefiles/unix.mk

```diff
@@ -38,6 +38,7 @@ test/helper:
 tempdir=$(shell ./bin/helper tempdir)
 test: test/helper build
 # 	Using `terramate` because it detects and fails if the generated files are outdated.
+	./bin/terramate run --no-recursive -- go test -c -tags interop ./cmd/terramate/e2etests/cloud/interop/...
```

Comment:
this must be reverted. The interop tests must be explicit invoked, as they are very slow and require proper environment set up.

Reply:
just noticed the `-c` in the `go test`. Why this is needed?

Reply:
since the interop tests have a build tag, they are not built  by `go test` and syntax errors won't be caught by the CI. 
`go test -c` only builds the test (it does not run them).